go.mod

```diff
@@ -33,6 +33,8 @@ require (
 	k8s.io/apiextensions-apiserver v0.18.9
 	k8s.io/apimachinery v0.18.9
 	k8s.io/client-go v0.18.9
-	sigs.k8s.io/cluster-api v0.3.8
+	sigs.k8s.io/cluster-api v0.3.10
 	sigs.k8s.io/controller-runtime v0.6.3
 )
+
+replace sigs.k8s.io/cluster-api v0.3.10 => github.com/giantswarm/cluster-api v0.3.10-gs
```

Comment:
This here was essentially the trick to get over with it. Thank you so much @giantswarm/team-celestial. 🌮